### DISCUSSION TITLE
Error while using REPL

### DISCUSSION THREAD

**Discussion by mastermunj** [adonisjs/core#2788] [Help] [answered]
I am facing an error while using REPL.

```sh
loadModels()
error TS6059: File 'REPL1' is not under 'rootDir' 'PROJECT_PATH'. 'rootDir' is expected to contain all source files.
  The file is in the program because:
    Root file specified for compilation
```

Any hints as to what could be wrong?

**Comment by thetutlage** [comment, marked as answer]
Please provide a sample repo to reproduce the error. Works fine for me on a brand new project with @adonisjs/lucid installed

**Reply by mastermunj** [reply]
I think the problem is related to typescript.

When new app is created using `adonis-ts-app`, typescript version is `4.2`. It works perfect in default setup.

However, when typescript is updated to `4.3.2`, the error starts.

I am not sure what could be causing this issue though.

**Reply by thetutlage** [reply]
Cool. Lemme try with `4.3.2`. Also TypeScript doesn't follow semver, so upgrading from `4.2` to `4.3` can introduce breaking changes

**Reply by thetutlage** [reply]
This is the commit that breaks the REPL. Let's see if they reply with any work around https://github.com/microsoft/TypeScript/commit/5813907abfe6ed757039383b4f33df3f9998112c

**Reply by mastermunj** [reply]
Awesome! Thank you for digging this.

**Reply by thetutlage** [reply]
Fix. https://github.com/adonisjs/require-ts/commit/9abddf0fe30a279995e9013d0146b17eac25ac10

**Reply by mastermunj** [reply]
That's super! I will test and give feedback.

**Reply by mastermunj** [reply]
It's working now. Thank you for quick fix.
